README.md

```diff
@@ -44,7 +44,8 @@ ipfs repo version | go-ipfs versions
                 4 |  0.4.3 - 0.4.5
                 5 |  0.4.6 - 0.4.10
                 6 |  0.4.11 - 0.4.15
-                7 |  0.4.16 - current
+                7 |  0.4.16 - 0.4.19
+                8 |  0.4.19 - current
```

Comment:
Depends if https://github.com/ipfs/go-ipfs/pull/6012 will make it to `0.4.19` release.